### ISSUE TITLE
Icon-only links/buttons need text

### ISSUE THREAD

**Issue by jvine** [sul-dlss/SearchWorks#350]
Next and Prev buttons have icons but no text content. 

```
<a class="btn btn-default btn-sm" href="/redesign/?f%5Baccess_facet%5D%5B%5D=At+the+Library&amp;page=2" rel="next" role="button">►</a>
```

should be:

```
<a href... ><span class="sr-only">Next </span><i class="fa fa-chevron-right"></i></a>
```

So the text version is in a hidden span (unless the Next is meant to display, in which case, no sr-only), and the icon is a font-icon, not an embedded symbol.

![screen shot 2014-06-26 at 5 22 17 pm](https://cloud.githubusercontent.com/assets/6945691/3406580/5950f748-fd92-11e3-9a65-12a6e64f30bc.png)
![screen shot 2014-06-26 at 5 22 10 pm](https://cloud.githubusercontent.com/assets/6945691/3406581/5952dbbc-fd92-11e3-9198-3f57ee45d243.png)
